src/graphql/mutations.js

```diff
@@ -323,6 +335,51 @@ export const addCreateCollectiveMutation = graphql(createCollectiveQuery, {
   }),
 });
 
+export const addCreateCollectiveFromGithubMutation = graphql(createCollectiveFromGithubQuery, {
+  props: ({ mutate }) => ({
+    createCollectiveFromGithub: async collective => {
+      const CollectiveInputType = pick(collective, [
+        'slug',
+        'type',
+        'name',
+        'image',
+        'description',
+        'longDescription',
+        'location',
+        'twitterHandle',
+        'githubHandle',
+        'website',
+        'tags',
+        'startsAt',
+        'endsAt',
+        'timezone',
+        'maxAmount',
+        'currency',
+        'quantity',
+        'HostCollectiveId',
+        'ParentCollectiveId',
+        'data',
```

Comment:
Which do we really expect or need from this list?

Reply:
Noted.